ScreenToGif/ImageUtil/ImageMethods.cs

```diff
@@ -1504,6 +1504,33 @@ public static Icon ToIcon(this ImageSource imageSource)
             }
         }
 
+        /// <summary>
+        /// Visits all the descendents of the visual using DFS and adds frozen copies 
+        /// of their drawing objects as children to the drawingGroup argument.
+        /// </summary>
+        /// <param name="visual">The visual to convert to a DrawingGroup</param>
+        /// <param name="drawingGroup">The target DrawingGroup to be populated</param>
+        static public void visualToFrozenDrawingGroup(this Visual visual, DrawingGroup drawingGroup)
```

Comment:
Hi, just a minor tweak in here, but I can do later:

`public static void VisualToFrozenDrawingGroup`

Reply:
Should be updated now. Sorry, we have an awful coding standard at 🏭 to which I am strongly habituated. I think I fixed the flicker as well.